event/narshe_wor.py

```diff
@@ -1,18 +1,24 @@
 from event.event import *
+from constants.checks import NARSHE_WEAPON_SHOP, NARSHE_WEAPON_SHOP_MINES
 
 class NarsheWOR(Event):
     def name(self):
-        return "Narshe WOR"
+        return NARSHE_WEAPON_SHOP.name
 
     def character_gate(self):
         return self.characters.LOCKE
 
     def init_rewards(self):
-        if self.args.no_free_characters_espers:
-            self.reward1 = self.add_reward(RewardType.ITEM)
-        else:
-            self.reward1 = self.add_reward(RewardType.ESPER | RewardType.ITEM)
-        self.reward2 = self.add_reward(RewardType.ITEM)
+
+        self.reward1 = self.add_reward(NARSHE_WEAPON_SHOP)
+        self.reward2 = self.add_reward(NARSHE_WEAPON_SHOP_MINES)
+
+        # This is to make sure you don't ever get an esper in the secondary slot and an item in the first.
+        # This ensures if either slot rolls an esper, the old man will pull one out.
+        if self.reward1.type == RewardType.ITEM:
```

Comment:
Add ability for narshe weapon shop to yield 2 espers ( i think this should be weighted against having 2 espers, maybe like 20% chance or something)